### ISSUE TITLE
Sonata form - thematic modules

### ISSUE THREAD

**Issue by bcmoseley** [openmusictheory/openmusictheory.github.io#26]
Needs an updated title on the table of contents page.

Examples for each of the thematic module types?

**Comment by kshaffer** [comment]
Agree. Or all of them can reference the same example piece (such as K. 333, which is already graphed on the exposition page).
